### PR TITLE
initial working version, needs style corrections

### PR DESCRIPTION
**Pull request by Buzunca**
This is assignment 2 for CSE 142.

### CODE REVIEW

File: assignment2/AsciiArt.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+public class AsciiArt {
```
Comment (euzuncao) [inline assignment2/AsciiArt.java]: Add header comment

Comment (Buzunca) [reply assignment2/AsciiArt.java]: good idea

File: assignment2/AsciiArt.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// CSE142
+// TA: Jun Song
+// Assignment #2 Part A
+//
+// This program will display ASCII art that I designed using for loops.
+
+public class AsciiArt {
+   public static void main(String[] args){
+      bar();
+      System.out.println("");
```
Comment (euzuncao) [inline assignment2/AsciiArt.java]: You can move this line to the bar() method, after the for loop.

File: assignment5/GuessingGame.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      }      
+
+      
+      public static void displayStats(int totalGuess, int numGame) {
+        double guessesPerGame = totalGuess / numGame;
+        
+          
+        System.out.println("Overall results:");
+        System.out.println("Total games = " + numGame);
+        System.out.println("Total guesses = " + totalGuess);
```
Comment (Buzunca) [inline assignment5/GuessingGame.java]: only shows the total number of guesses for the first game, not all games combined.

File: assignment5/GuessingGame.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        System.out.println("Guesses/game = " + guessesPerGame);
+        //System.out.println("Best game = " + minGuess);
```
Comment (Buzunca) [inline assignment5/GuessingGame.java]: not sure how to find the minimum guesses of all games. I know i need to use math class

File: assignment5/GuessingGame.java (hunk 3 above)
Comment (euzuncao) [reply assignment5/GuessingGame.java]: numGame and totalGuess need to be global variables, class level. If they are defined inside the method then they will only hold the values for that game.

File: assignment5/GuessingGame.java (hunk 4 above)
Comment (euzuncao) [reply assignment5/GuessingGame.java]: minGuess also needs to be a global variable. Every time you call displayStats with numGuess, you update minGuess if it's smaller.

File: assignment5/GuessingGame.java [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+public class GuessingGame {
+   //This is a class constant for the set minimum integer of the range.
+   public static final int MIN = 1;
+   //This is a class constant for the set maximum integer of the range.
+   public static final int MAX = 10;
+   
+   public static void main(String[] args) {
+   
+   Scanner console = new Scanner(System.in);
```
Comment (euzuncao) [inline assignment5/GuessingGame.java]: INdentation seems to be wrong

File: assignment5/GuessingGame.java [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+   
+   int totalGuess = playGame(console);
+   int numGame = playAgain(console, totalGuess);   
+   
+   }
+      
+      
+      public static void introHaiku() {
```
Comment (euzuncao) [inline assignment5/GuessingGame.java]: What is this?

File: assignment5/GuessingGame.java [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+      }
+      
+      public static int playAgain(Scanner console, int numGuess) {
+         int numGame = 1;
+                  
+         System.out.println("Do you want to play again? ");
+         String playAgain = console.next();
+         playAgain = playAgain.toLowerCase();
+         
+         while (playAgain.startsWith("y")) {
```
Comment (euzuncao) [inline assignment5/GuessingGame.java]: while(playAgain())
{...}

**Pull request merged**